display/column.go

```diff
@@ -72,7 +72,7 @@ func ColumnValueAsString(val interface{}, colType *sql.ColumnType) (result strin
 // JSON, JSONB, BOOL and so on..
 func ParseJSONOutputColumnValue(val interface{}, colType *sql.ColumnType) (interface{}, error) {
 	if val == nil {
-		return constants.NullString, nil
+		return nil, nil
```

Comment:
why do we remove this? We're still displaying the constants.NullString though I presume?

Reply:
If we don't remove this, then we will get something like:
```
{
  "value": "<null>"
}
```
as opposed to:
```
{
  "value": null
}
```

Reply:
hmm so the null character is used for table, line and csv, but json uses actual nulls?